R/plotBoxplot.R

```diff
@@ -480,7 +555,7 @@ setMethod("plotBoxplot", signature = c(object = "SummarizedExperiment"),
     # first instance. Otherwise it would be time point 1 -> time point 2,
     # which is NA.
     df <- df |>
-        arrange(desc(across(all_of(c(pair.by, x)))))
+        arrange(desc(across(all_of(c("difference", pair.by, x)))))
```

Comment:
This was added to ensure that the line between paired samples was colored correctly.